practice/00-HTML-CSS-basics/ExerciseOne.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>Botcamp Practice</title>
+	</head>
+<body>	
+	<header>
```

Comment:
A header must contains different headings (h1 - h6), a logo or any other info that's part of the header of a page or site. Things like the list do not belong here.